File: Python/card.py
from Python.payment import Payment

class Card(Payment):
    cardName = str
    cardNumber = int
    ccv = int
    expirationDate = int

    def __init__(self, cardName, cardNumber, ccv, expirationDate):
        super().__init__(id)
        self.cardName = cardName
        self.cardNumber = cardNumber
        self.ccv = ccv
        self.expirationDate = expirationDate

File: Java/Card.java
public class Card extends Payment {
    Integer id;
    String cardName;
    Integer cardNumber;
    Integer ccv;
    Integer expirationDate;

    public Card(Integer id, String cardName, Integer cardNumber, Integer ccv, Integer expirationDate) {
        super(id);
        this.cardName = cardName;
        this.cardNumber = cardNumber;
        this.ccv = ccv;
        this.expirationDate = expirationDate;
    
}
}
